after/queries/typescript/highlights.scm

```diff
@@ -0,0 +1,12 @@
+;; extends
+[
+("=>" @operator.fat_arrow (#set! "priority" 1000))
+]
+
+[
+(enum_declaration (enum_body (property_identifier) @property.enum) (#set! "priority" 1000))
+]
+
+[
+(interface_declaration (object_type (property_signature (property_identifier) @property.function (type_annotation (type_identifier) @type) (#eq? @type "FuncType"))))
+]
```

Comment:
What is FuncType in TS? `#eq` captures the literal text

Reply:
<img width="434" alt="Screen Shot 2023-03-04 at 09 43 29@2x" src="https://user-images.githubusercontent.com/9512444/222891834-27cdb9ff-8ecd-4cff-a985-a5c61a359cbb.png">

For this purpose, it was about highlighting a property with function hlgroups if it's a function type. This was a nice visual differentiation that Visual Studio Code includes

Reply:
Is FuncType a builtin identifier to say that a property is a function type?